app/models/team.rb

```diff
@@ -45,18 +45,23 @@ def visible
     end
 
     # Returns a base scope reflecting the relevant teams depending on what
-    # phase of the running season we're currently in.
+    # phase of the running season we're currentin_ly in.
     def by_season_phase
       if Time.now.utc > Season.current.acceptance_notification_at
-        Team.where season: Season.current, kind: %w(sponsored voluntary)
+        Team.in_current_season.selected
       else
-        Team.visible.where season: Season.current
+        Team.in_current_season.visible
       end
     end
 
-    def active_teams
-      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
+    def in_current_season
```

Comment:
The concrete name »current« is not a Rails convention, but re-using method names on different classes (when the expected behaviour is the same or at least similar) is a _Ruby_ convention. Ruby focusses heavily on the idea of _duck typing_ – meaning it doesn't really care _what_ it is as long as it responds to a certain method.

But I understand what you mean, and the name `current` is totally up for debate.

What I suggest: let's stick with `current` in this PR. A follow-up PR can refactor all definitions of `current` (and its relevant factory traits) to `in_current_season`. It's a longer method name, but it totally reveals its intention. And that's way better! What do you think?